public/index.html

```diff
@@ -28,7 +28,7 @@
         <div class="container">
             <div class="row">
                 <div class="col-md-12">
-                   &copy; 2017 TokenMint.io |<a href="http://www.tokenmint.io/about" target="_blank"> About Us </a> | <i class="fa fa-github" aria-hidden="true"></i> | <i class="fa fa-twitter" aria-hidden="true"></i>
+                   &copy; 2017 TokenMint.io |<a href="https://github.com/ethereumproject/TokenMint/wiki" target="_blank"> About Us </a> | <a href src="https://github.com/ethereumproject/TokenMint"><i class="fa fa-github" aria-hidden="true"></i></a> | <a href src="https://twitter.com/theTokenMint"><i class="fa fa-twitter" aria-hidden="true"></i></a>
```

Comment:
no we need one. :) @tokenmint already exists so I just added @theTokenmint as a place holder for now. @ChuckSRQ